### DISCUSSION TITLE
Trouble Binding Local Directories in Docker-Compose

### DISCUSSION THREAD

**Discussion by tpups** [lloesche/valheim-server-docker#376] [Q&A] [answered]
I have the container running in an ubuntu 20.04 VM. I'm able to browse to the server, but I can't seem to get the binding to work to move my world over. I did briefly get it working by manually copying the files into the container, but it seems to disappear on restart.

Specifically, these two lines in docker-compose.yaml don't seem to work:

`    volumes: `
`      - $HOME/valheim-server/config:/config`
`      - $HOME/valheim-server/data:/opt/valheim`

I copied my world files into $HOME/valheim-server/config/worlds and set the world name to match in valheim.env. However, the server just generates a new map with that name.

Also, I'm not entirely clear what the /data directory is for. I don't have any files to put there?

**Comment by tpups** [comment, marked as answer]
Was able to get it working by adding single quotes and changing the path to a relative path:

`volumes:`
`- './config:/config'`

Additionally I removed the /opt/volume mount as I don't have the server downloaded outside the container.

**Reply by lloesche** [reply]
Without the data directory the container will re-download the Valheim server on every fresh restart.
